sanic/sanic.py

```diff
@@ -1,5 +1,9 @@
-import asyncio
+from argparse import ArgumentParser
+from asyncio import get_event_loop
 from inspect import isawaitable
+from multiprocessing import Process, Event
```

Comment:
I think the multiprocess function should not contain in the program, just let user to decide use multi process or not